bot/commands/command_logic/pomo.py

```diff
@@ -121,39 +134,54 @@ def __get_break_start_text(self):
 
 __pomo_users: Dict[str, PomoTimerThread] = {}
 
+# public methods
+
 
-async def handle_pomo(ctx: Message):
+async def handle_pomo(ctx: Message) -> None:
     args: List[str] = __get_message_args(ctx.content)
     username = ctx.author.name
 
     current_pomo = __pomo_users.get(username)
 
-    if current_pomo:
-        if len(args) and args[0] == 'cancel':
-            __cancel_pomo(username)
+    if len(args) == 0:
+        if current_pomo:
+            await __show_pomo_update(username, current_pomo, ctx)
```

Comment:
you have `username` on the `PomoTimerThread` object already, right? so no need to pass it in here?

also `__pomo_users` is kinda misleadingly named, would call it `__pomo_timers` or something along those lines